### PR TITLE
Update Favicon to the new Brave logo (CU-8678w897a)

### PR DESCRIPTION
**Pull request by tmackinnon**
PA favicon was old Brave logo, updated the file with new Brave logo. 

Test Plan: 
- [x] Check that correct logo appears on localhost
- [x] Deploy to dev and visually check that correct logo appears
- [x] Make sure Travis passes

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-8678w897a Update Favicon in PA to the new Brave logo](https://app.clickup.com/t/8678w897a)

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ the code was deployed.
 
 ## [Unreleased]
 
+### Changed
+- Updated favicon image from old logo to current logo
```
Comment (micahdbak) [inline CHANGELOG.md:15]: Good!

**Review by micahdbak — APPROVED** [review]
Yay consistency!

File: CHANGELOG.md (hunk 1 above)
Comment (johnny-brave) [reply CHANGELOG.md:15]: Not very important for this deployment, but we should include the clickup task ID and end the line with a period for consistency.

Comment (tmackinnon) [reply CHANGELOG.md:15]: Ah good catch - thanks Johnny! I'll add it whenever we deploy to Production next.

**Pull request merged**